plugins/azure-devops/src/routes.ts

```diff
@@ -13,8 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import { createRouteRef } from '@backstage/core-plugin-api';
 
-export const azureDevOpsRouteRef = createRouteRef({
-  title: 'azure-devops',
+export const azurePrsRootRouteRef = createRouteRef({
```

Comment:
Can we use `azurePullRequestDashboardRouteRef` here instead? That will help make things more explicit between the Dashboard and Entity routes

Reply:
Good idea. Updated.